.github/workflows/ci.yml

```diff
@@ -71,8 +71,12 @@ jobs:
       run: ./bootstrap.sh
     - name: Run v3 Tests (Debug)
       run: bundle exec fastlane test
-    - name: Post Codecov report
-      run: bash <(curl -s https://codecov.io/bash) -t ${{ secrets.CODECOV_TOKEN }}
+    - uses: codecov/codecov-action@v1
+      with:
+        token: ${{ secrets.CODECOV_TOKEN }}
+        flags: unittests-ios14
+        fail_ci_if_error: true # if codecov upload fails, should fail ci
```

Comment:
How often does the upload fail? :D

Reply:
I don't know actually 😄 But they say this can happen randomly 🤷‍♂️  These unexpected changes are hard to debug, so better to fix every variable we can. Even with this PR, I'm not yet 100% this will fix it.

Reply:
fair enough, we can merge and see

Reply:
Okay, let's merge it then 👍 I'll add the UI tests in the Clean CI ticket I'm currently working on